ui/src/app/settings/components/UserForm/UserForm.test.js

```diff
@@ -0,0 +1,10 @@
+import { shallow } from "enzyme";
+import React from "react";
+import { UserForm } from "./UserForm";
+
+describe("UserForm", () => {
+  it("can render", () => {
+    const wrapper = shallow(<UserForm title="Add user" />);
+    expect(wrapper).toMatchSnapshot();
+  });
+});
```

Comment:
UserForm seems to have a lot of logic that we aren't testing. I suppose we want to avoid testing Formik itself, but perhaps we should validate that some of the form's expected behaviours?